### PR TITLE
python/two-fer: Variadic Argument List

### PR DESCRIPTION
**Pull request by jeffdparker**
Deal with an alternative solution.

```python
def two_fer(*name):
    if name:
        for var in name:
            return "One for "+ var + ", one for me."
    else:
        return 'One for you, one for me.'
```

### CODE REVIEW

File: tracks/python/exercises/two-fer/mentoring.md [hunk 1]
```diff
@@ -76,3 +76,20 @@ def two_fer(name=None):
...
         return 'One for you, one for me.'
 ```
+
+#### Variadic Argument List
+
+We can use a variadic argument list to solve the problem
+
+```python
+def two_fer(*name):
+    if name:
```
Comment (cmccandless) [inline tracks/python/exercises/two-fer/mentoring.md]: `if` statement is redundant. The following is functionally equivalent:

```Python
def two_fer(*name):
  for var in name:
    return f'One for {var}, one for me.'
  else:
    return f'One for you, one for me.'
```

File: tracks/python/exercises/two-fer/mentoring.md [hunk 2]
```diff
@@ -76,3 +76,20 @@ def two_fer(name=None):
...
+        return 'One for ' + name + ', one for me.'
+    else:
+        return 'One for you, one for me.'
+```
+
+This satisfies the tests, but it more complex than needed.
+Encourage the the student to find a simpler solution:
```
Comment (cmccandless) [inline tracks/python/exercises/two-fer/mentoring.md]: :+1:

File: tracks/python/exercises/two-fer/mentoring.md (hunk 1 above)
Comment (jeffdparker) [reply tracks/python/exercises/two-fer/mentoring.md]: You are right.  I pasted in what the student wrote, and was regretting it while rereading this morning.

Comment (cmccandless) [reply tracks/python/exercises/two-fer/mentoring.md]: @jeffdparker You marked this as resolved, but did not commit any changes or state an intent to leave as-is.

Comment (jeffdparker) [reply tracks/python/exercises/two-fer/mentoring.md]: Your change seems to be in place - When I review changes, the approve button is not enabled.

Comment (jeffdparker) [reply tracks/python/exercises/two-fer/mentoring.md]: Honestly, I have no memory of doing the right thing.

**Comment by cmccandless** [comment]
Apologies all; I had approved this several days ago but never assigned back to @SleeplessByte for merge.

**Comment by SleeplessByte** [comment]
Thanks!

**Pull request merged**